modules/vba_documents/spec/models/upload_submission_spec.rb

```diff
@@ -23,6 +23,12 @@
            status: 'error', code: 'DOC202',
            detail: 'Upstream status: Errors: ERR-EMMS-FAILED, Corrupted File detected.')
   end
+  let(:upload_error_upstream_old) do
+    create(:upload_submission,
+           created_at: 20.days.ago,
+           status: 'error', code: 'DOC202',
+           detail: 'Upstream status: Errors: ERR-EMMS-FAILED, Corrupted File detected.')
+  end
```

Comment:
This looks like an opportunity to use the new factory trait that you created above:
```
let(:upload_error_upstream_old) do
    create(:upload_submission, :status_upstream_error, created_at: 20.days.ago))
end
```

Reply:
done